.storybook/main.ts

```diff
@@ -4,6 +4,10 @@ import {mergeConfig} from "vite";
 
 import type {StorybookConfig} from "@storybook/react-vite";
 
+function getAbsolutePath(value: string): any {
+    return dirname(require.resolve(join(value, "package.json")));
+}
```

Comment:
I think this `getAbsolutePath` function is unused.

Reply:
🤦‍♂️ You're right. While trying to figure out the issue I pulled in this function as it was also in the WB config (they're on Storybook 8 though). I'll clean this up. Thanks for catching that.